react/src/components/KeypairResourcePolicyList.tsx

```diff
@@ -110,6 +110,7 @@ const KeypairResourcePolicyList: React.FC<KeypairResourcePolicyListProps> = (
       key: 'name',
       fixed: 'left',
       sorter: (a, b) => localeCompare(a?.name, b?.name),
+      width: 200,
```

Comment:
Is it intended? If you want to reflect resizable property, making a child stack is better.

Reply:
I wrote the actual use case for ease of testing, but as you said, it seems better to have the application part on a separate stack. I'll comment that out.